backend/module-campaigns/src/Podkrepibg.Campaigns.Infrastructure/Configuration/DbConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+namespace Podkrepibg.Campaigns.Infrastructure.Configuration
+{
+    using System;
+
+    public class DbConfiguration
+    {
+        private static string DbHost = Environment.GetEnvironmentVariable("DB_HOST");
```

Comment:
You should get those from the `Configuration`, not the `Environment`.

Reply:
See [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-5.0#environment-variables). It will also make testing significantly easier as well as we can then mock the configuration, load it from a file.

Reply:
> You should get those from the `Configuration`, not the `Environment`.

In the story is written that the string values should be read from env : 
The connection string needs to be created from the DB_HOST and DB_PORT coming from .env files.

Reply:
yep, I also support using the configuration infrastructure, it will be more flexible that way - for example we can override db_port via a commandline argument if we want to. You just need to add a `EnvironmentVariablesConfigurationProvider`.

Reply:
Now configuration.GetSection is used.